wpilibcExamples/src/main/cpp/examples/ArcadeDriveXboxController/cpp/Robot.cpp

```diff
@@ -0,0 +1,35 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include <frc/GenericHID.h>
+#include <frc/PWMVictorSPX.h>
+#include <frc/TimedRobot.h>
+#include <frc/XboxController.h>
+#include <frc/drive/DifferentialDrive.h>
+
+/**
+ * This is a demo program showing the use of the DifferentialDrive class.
+ * Runs the motors with arcade steering and an Xbox controller.
+ */
+class Robot : public frc::TimedRobot {
+  frc::PWMVictorSPX m_leftMotor{0};
+  frc::PWMVictorSPX m_rightMotor{1};
+  frc::DifferentialDrive m_robotDrive{m_leftMotor, m_rightMotor};
+  frc::XboxController m_driverController{0};
+
+ public:
+  void TeleopPeriodic() {
+    // Drive with arcade style
```

Comment:
Technically isn't this split arcade style (since Y is left, X is right)?  Maybe like tank, make this clear in the comment?

Reply:
Oh, I think it should be split arcade, what is the configuration for that?

Reply:
I think Peter just wants you to clarify in the comments that you are using a split-arcade style instead of traditional arcade.

Reply:
Ah, ok will fix that. Thanks!